remote_states.tf

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------------------------------------------------
+# Retrieves state data from a Terraform backend. This allows use of
+# the root-level outputs of one or more Terraform configurations as
+# input data for this configuration.
+# ------------------------------------------------------------------------------
+
+data "terraform_remote_state" "master" {
+  backend = "s3"
+
+  config = {
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+    key            = "cool-accounts/master.tfstate"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+  }
+
+  # There is only one environment for this account, so there is
+  # no need to match the current Terraform workspace.
+  workspace = "production"
+}
+
+data "terraform_remote_state" "users" {
+  backend = "s3"
+
+  config = {
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+    key            = "cool-accounts/users.tfstate"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+  }
+
+  # There is only one environment for this account, so there is
+  # no need to match the current Terraform workspace.
+  workspace = "production"
+}
+
+data "terraform_remote_state" "userservices_was_db_staging" {
+  backend = "s3"
+
+  config = {
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+    key            = "cool-userservices-was-db/terraform.tfstate"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+  }
+
+  workspace = "staging"
+}
+
+data "terraform_remote_state" "userservices_was_db_production" {
+  backend = "s3"
+
+  config = {
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+    key            = "cool-userservices-was-db/terraform.tfstate"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
```

Comment:
Alphabetize.

```suggestion
    bucket         = "cisa-cool-terraform-state"
    dynamodb_table = "terraform-state-lock"
    encrypt        = true
    key            = "cool-userservices-was-db/terraform.tfstate"
    profile        = "cool-terraform-backend"
    region         = "us-east-1"
```

Reply:
I'm going to resolve this conversation, since the suggestion is the same as what is already there. 😄 Please re-open if I'm missing something here.